src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/StaffRefDataController.java

```diff
@@ -414,5 +416,44 @@ public ResponseEntity<StaffProfileCreationResponse> updateStaffUserProfile(@Requ
         return ResponseEntity.status(HttpStatus.OK).body(staffProfileCreationResponse);
     }
 
+    @ApiOperation(
+            value = "This API search a staff user by Id",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
```

Comment:
Hi @sahitya-desireddy just make The Swagger Documentation as per the code . Here the response code should be match with the code (200).

Reply:
Thanks @manukundloo-hmcts resolved now please review